runtime/vm/MHInterpreter.cpp

```diff
@@ -555,6 +555,12 @@ VM_MHInterpreter::dispatchLoop(j9object_t methodHandle)
 			/* Get MethodHandle for this operation from the VarHandle's handleTable */
 			j9object_t handleTable = J9VMJAVALANGINVOKEVARHANDLE_HANDLETABLE(_currentThread, varHandle);
 			j9object_t methodHandleFromTable = J9JAVAARRAYOFOBJECT_LOAD(_currentThread, handleTable, operation);
+
+			if (NULL == methodHandleFromTable) {
```

Comment:
This still needs a methodtype frame to be built as we don't know what MH operations have lead to this point.  The stack needs to be kept walkable here

Reply:
Re-added `buildMethodTypeFrame`.